frontstage/views/account/account_transfer_survey.py

```diff
@@ -247,21 +205,24 @@ def transfer_survey_post_email_entry(session):
 @jwt_authorization(request)
 def send_transfer_instruction_get(session):
     email = flask_session["transfer_survey_recipient_email_address"]
-    share_dict = {}
-    for business_id in flask_session["transfer_survey_data"]:
-        selected_business = get_business_by_id(business_id)
+    transfer_dict = {}
+    for business in flask_session["transfer_survey_data"]:
+        flask_session.pop("transfer_surveys_selected_list", None)
```

Comment:
Not a fan of using flask_session, any other options?

Reply:
For the moment I don't want to touch this method of storing. I'd feel better with this were put into another card.

Reply:
As discussed, I think its ok to store what is effectively transient/interim UI state in the flask session. The alternatives are passing it as hidden form data, query string parameters, or even persisting it in the DB. However, it looks to me like we have the server side flask session and the session cookie the wrong way round with the cookie holding the data state and the session ID token in Redis. But that's how it's been designed so we won't change that here.